### DISCUSSION TITLE
There is no hunt and optionally pivot to PCAP on 2.4 anymore ?

### DISCUSSION THREAD

**Discussion by majungman** [Security-Onion-Solutions/securityonion#12027] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

20

### RAM

100000

### Storage for /

1000

### Storage for /nsm

1000

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

hi , anyone knows how to get the " hunt and optionally pivot to PCAP " on 2,4 version??? 

this is 2.3 version 
<img width="236" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/db4d40de-f5c9-4f8a-99e5-6bd327ef1e90">

with 2.4 version i see the id only , which it wont click and direct to the PCAP 

<img width="223" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/0f5ad00d-b8ab-41d1-ac73-165ef93af09c">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by majungman** [comment]
any ideas how can i config that ?

**Comment by dougburks** [comment]
As mentioned at https://github.com/Security-Onion-Solutions/securityonion/discussions/11905:

You should be able to do something similar by copying the log.id.uid field value and then searching SOC Dashboards or Hunt for it.

Most folks avoid this issue altogether by simply using SOC Dashboards as their primary dashboards interface:
https://docs.securityonion.net/en/2.4/dashboards.html

We have no plans to update the Kibana dashboards.

Is there some particular reason why you can't use SOC Dashboards?

If you absolutely have a hard requirement for Kibana, you could always create your own custom Kibana dashboards.
